src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/LagPortOperationService.java

```diff
@@ -213,45 +219,94 @@ public String getSwitchIpAddress(SwitchId switchId) throws InvalidDataException,
                         format("Switch %s has invalid IP address %s", sw, sw.getSocketAddress())));
     }
 
-    private LagLogicalPort createTransaction(SwitchId switchId, Set<Integer> targetPorts)  {
-        Switch sw = querySwitch(switchId); // locate switch first to produce correct error if switch is missing
-        if (! isSwitchLagCapable(sw)) {
-            throw new InvalidDataException(format("Switch %s doesn't support LAG.", sw.getSwitchId()));
-        }
-
-        Set<SwitchFeature> features = sw.getFeatures();
-        for (Integer portNumber : targetPorts) {
-            validatePhysicalPort(switchId, features, portNumber);
+    private void verifyTargetPortsInput(Set<Integer> targetPorts) {
+        if (targetPorts == null || targetPorts.isEmpty()) {
+            throw new InvalidDataException("Physical ports list is empty");
         }
+    }
 
-        ensureNoLagCollisions(switchId, targetPorts);
+    private LagLogicalPort createTransaction(SwitchId switchId, Set<Integer> targetPorts)  {
```

Comment:
Extra space.

Reply:
hm... why checkstyle do not points on it?.. fixed.